assets/css/_invalid_notification_modal.scss

```diff
@@ -0,0 +1,42 @@
+.m-invalid-notification-modal__close-button {
+  height: 1rem;
+  position: relative;
+
+  button {
+    position: absolute;
+    right: 0;
+
+    svg {
+      height: 1rem;
+    }
+  }
+}
+
+.m-invalid-notification-modal__body {
+  margin-top: 1rem;
+}
+
+.m-invalid-notification-modal__buttons {
+  display: flex;
+  height: 1rem;
+  margin-top: 1.5rem;
+}
+
+.m-invalid-notification-modal__button-wrapper {
```

Comment:
To get the buttons centered, instead of `position: absolute`ing them to the sides of a wrapper, I think you can remove the wrappers with:
```
.buttons {
  display: flex;
  justify-content: center
}

.keep-button {
  flex: 0 1 auto; // the 0 prevents it from growing above its size
  width: 7rem;
}

.discard-button {
  // same
}